### ISSUE TITLE
Headings are big

### ISSUE THREAD

**Issue by mkrtchian** [LeonStaufer/material-dokuwiki#47]
There are a few todos in the CSS. This issue is for [heading's size](https://github.com/LeonStaufer/material-dokuwiki/blob/master/css/basic.less#L301). Right now they are quite big.

**Comment by LeonStaufer** [comment]
Good point, I'd suggest following the [Material spec](https://material.io/design/typography/the-type-system.html#type-scale) and I could quickly add a commit for that.
